spec/index.html

```diff
@@ -3499,9 +3531,19 @@ <h2>A Canonical form of N-Quads</h2>
       and <code>graphLabel</code>,
       each of which MUST be a single space (<code>U+0020</code>).</li>
     <li><a data-cite="RDF11-CONCEPTS#dfn-literal">Literals</a> with the
-      datatype <code>http://www.w3.org/2001/XMLSchema#string</code>
+      datatypes <code>http://www.w3.org/2001/XMLSchema#string</code> and
+      <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#langString</code>
       MUST NOT use the datatype IRI part of the <a data-cite="N-QUADS#grammar-production-literal">literal</a>,
-      and are represented using only <a data-cite="N-QUADS#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a>.
+      and are represented using only <a data-cite="N-QUADS#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a>
+        and <a href="N-QUADS#grammar-production-LANGTAG">LANGTAG</a>,
+        as appropriate.
+        <div class="issue" data-number="155">
+          [[RDF11-CONCEPTS]] allows implementations to convert
+          <a data-cite="RDF11-CONCEPTS#dfn-language-tag">language tags</a>.
+          This represents a potential interoperability issue if language tags
+          are not converted to lower case uniformly, as different implementations
+          might end up using different canonical representations of such language tags.
+        </div>
```

Comment:
This is a matter of potentially normalizing the language tag on input, not on output.

```suggestion
      and are represented using only <a data-cite="N-QUADS#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a>.
```